brands/amenity/cinema.json

```diff
@@ -347,6 +347,17 @@
       "name": "The Space Cinema"
     }
   },
+  "amenity/cinema|UGC": {
+    "countryCodes": ["fr"],
+    "tags": {
+      "amenity": "cinema",
+      "brand": "UGC",
+      "brand:website": "http://ugc.fr",
```

Comment:
Can you remove this tag?  We don't include `brand:website` in the index because brands can change it.  Anyone who wants this data should probably get the current website from the brand's wikidata page.